docs/Parameters.rst

```diff
@@ -1231,7 +1233,8 @@ Query Data
 ~~~~~~~~~~
 
 For learning to rank, it needs query information for training data.
-LightGBM uses an additional file to store query data, like the following:
+
+LightGBM's CLI uses an additional file to store query data, like the following:
```

Comment:
Not necessary `CLI`. Wrappers (at least Python one) also can use this feature when `data` param is filename.
```suggestion
LightGBM's uses an additional file to store query data, like the following:
```
https://github.com/microsoft/LightGBM/blob/706f2af7badc26f6ec68729469ec6ec79a66d802/python-package/lightgbm/basic.py#L1125-L1132

https://github.com/microsoft/LightGBM/blob/706f2af7badc26f6ec68729469ec6ec79a66d802/src/c_api.cpp#L913

https://github.com/microsoft/LightGBM/blob/706f2af7badc26f6ec68729469ec6ec79a66d802/src/io/dataset_loader.cpp#L204

https://github.com/microsoft/LightGBM/blob/f30dbe8717177da2b1745ac3b2a5d5fd7475de55/src/io/metadata.cpp#L26


https://github.com/microsoft/LightGBM/blob/f30dbe8717177da2b1745ac3b2a5d5fd7475de55/src/io/metadata.cpp#L435-L443

Reply:
ohhhh I misunderstood

Reply:
```suggestion
LightGBM uses an additional file to store query data, like the following:
```